packages/zoe/src/contracts/sellItems.js

```diff
@@ -60,15 +60,16 @@ const start = zcf => {
       throw buyerSeat.kickOut(rejectMsg);
     }
 
+    // All items are the same price.
     const totalCost = moneyMath.make(
       pricePerItem.value * wantedItems.value.length,
     );
 
     // Check that the money provided to pay for the items is greater than the totalCost.
-    if (!moneyMath.isGTE(providedMoney, totalCost)) {
-      const rejectMsg = `More money (${totalCost}) is required to buy these items`;
-      throw buyerSeat.kickOut(rejectMsg);
-    }
+    assert(
+      moneyMath.isGTE(providedMoney, totalCost),
+      details`More money (${totalCost}) is required to buy these items`,
```

Comment:
Is `totalCost` sensitive, or do you want to reveal it up the call chain? The old code revealed it. The new code hides it from callers and reveals it on the console. The upcoming assert/error/console reform will do a better job of making this debugging info visible to the developer while keeping it hidden from callers, so this change is probably good long term.

However, if you do not intend to hide the cost from callers, do:

```suggestion
      details`More money (${q(totalCost)}) is required to buy these items`,
```

Reply:
Hmm, I think it's not particularly sensitive, in that it is the cost per item (which is public) * the number of items really the only new information is the number of items that the caller is trying to buy. I guess it might be sensitive enough to not deliberately expose it. I think I'll keep this as is, if that's ok